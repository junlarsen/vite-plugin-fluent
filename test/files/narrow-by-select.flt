narrow = This has been seen { $count } times, so we select by { $count ->
  [one] one
 *[other] other
} and it should narrow count down to one or other.