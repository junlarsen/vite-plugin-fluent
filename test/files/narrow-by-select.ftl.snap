import type { FluentResource, FluentBundle, FluentVariable } from "@fluent/bundle";
export declare const resource: FluentResource;
export declare function formatMessage(bundle: FluentBundle, id: "narrow", args: {
    readonly "count": "one" | "other";
}, error?: null | Error[]): string;
